web/src/main/java/io/fabric8/launcher/web/endpoints/GitEndpoint.java

```diff
@@ -72,7 +71,7 @@ public GitUser getUser() {
     @Produces(MediaType.APPLICATION_JSON)
     public Set<String> getRepositories(@QueryParam("organization") String organization) {
         ImmutableGitOrganization org = organization != null ? ImmutableGitOrganization.of(organization) : null;
-        return gitService.getRepositories(org).stream()
+        return gitService.getRepositories(ImmutableGitRepositoryFilter.of(org)).stream()
```

Comment:
Is the org always necessary for the creation of ImmutableGitRepositoryFilter? See my comment about @Parameter

Reply:
Ok the singleton should answer you question.